contents/blog/how-to-measure-product-market-fit.md

```diff
@@ -0,0 +1,332 @@
+---
+date: 2023-06-08
+title: "In-depth: How to reliably measure product-market fit"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/pmf-game-guide/pmf-guide.png
+featuredImageType: full
+category: Startups
+tags:
+  - Guides
+---
+
+[Achieving product-market fit](/blog/product-market-fit-game) isn't a golden ticket to a billion-dollar IPO, but startups can't succeed without it. 
+
+Most guides tell you how it _feels_ to have product-market fit. Everything feels effortless when you have it, or impossibly difficult when you don't, and so on.
+
+But product-market fit isn't just an ephemeral gut feeling. You can _measure_ it, and _it moves_ as your customer's needs change, or new solutions appear.
+
+This guide is about finding ways to measure product-market fit, your progress toward it, and how it changes over time.
+
+## Leading and lagging indicators
+
+Measuring product-market fit requires a _combination_ of leading and lagging indicators.
+
+**Leading indicators**, are predictive. A surge in organic growth and positive word-of-mouth suggests product-market fit, but can't confirm it. They include:
+- **Organic word-of-mouth growth** – Not directly linked to paid promotions
+- **Meaningful user engagement** – Users experiencing real value    
+- **User feedback & PMF Survey** – Direct feedback from users
+
+**Lagging indicators** confirm whether you have product-market fit. They're more reliable, particularly if they're linked to financial performance. They include:
+- **Retention curve** – Evidence users are coming back
+- **Burn multiple** – Evidence you're growing sustainably
+- **CLV/CAC ratio** – Can you acquire customers for less than they cost to support? 
+
+We recommend a minimum of three to start with – likely two leading indicators and one lagging indicator – and adding more when you need.
+
+**Pre-revenue**, word-of-mouth, user engagement, and retention are good indicators of product-market fit. They'll tell you how people feel about your product, whether they're using it, and if they stick around.
+
+**Post-revenue** you'll want to want another lagging indicator, like burn multiple or CLV/CAC ratio, that provide a more concrete link to revenue and the financial viability of your product.
+
+Why a minimum of three? Because of what Brian Balfour, a serial co-founder and a former VP of Growth at Hubspot, calls The Trifecta:[^1]
+
+![pmf trifecta](../images/blog/how-to-measure-product-market-fit/pmf-trifecta.png)
+
+If you can nail these three at the same time, it's a strong sign you have product-market fit.
+
+## Indicator #1: Organic word-of-mouth growth
```

Comment:
Really enjoyed this section on the different ways of measuring word of mouth growth! Very actionable